app/src/pages/common/settingsPage/patternAnalysisTab/patternList/patternList.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { PatternListItem } from './patternListItem';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
Which purpose of it? For future development?

Reply:
Yes, it was for future usage. But for now I decided extract logic in other component. So, I 've just written this as functional component.